src/OneExplorer/OneExplorer.ts

```diff
@@ -261,8 +261,7 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<OneNode> {
         .then(ans => {
           if (ans === approval) {
             Logger.info('OneExplorer', `Delete '${oneNode.node.name}'.`);
-            return vscode.workspace.fs.delete(
-                oneNode.node.uri, {recursive: recursive, useTrash: true});
+            vscode.workspace.fs.delete(oneNode.node.uri, {recursive: recursive, useTrash: true});
```

Comment:
OR, 

```suggestion
            vscode.workspace.fs.delete(oneNode.node.uri, {recursive: recursive, useTrash: true}).then(()=>this.refresh());
```

This looks better then adding a resolved Promise<void>